MetadataProcessor.Shared/Tables/nanoSignaturesTable.cs

```diff
@@ -587,6 +686,20 @@ private byte[] GetSignature(
             }
         }
 
+        private byte[] GetSignature(Collection<GenericParameter> genericParameters)
+        {
+            using (var buffer = new MemoryStream())
+            using (var writer = new BinaryWriter(buffer)) // Only Write(Byte) will be used
```

Comment:
minimal: rather than the 2 using in 2 lines, maybe use the brackets as well? (super minimal)